src/main/java/edu/wpi/first/pathweaver/ProjectPreferences.java

```diff
@@ -236,7 +236,7 @@ public Values getValues() {
 		private String exportUnit;
 		private final double maxVelocity;
 		private final double maxAcceleration;
-		private final double wheelBase;
+		private final double trackWidth;
```

Comment:
This breaks save file backwards compatibility. We need to "upgrade" user save files.